lib/services/item_manager.ts

```diff
@@ -142,7 +142,7 @@ export class ItemManager extends PureService {
   /**
    * Returns all non-deleted items keys
    */
-  get itemsKeys() {
+  itemsKeys() {
```

Comment:
Did we decide not to rename this `latestItemsKeys`?

Reply:
We made that decision for the `protocol_service.ts` method: https://github.com/standardnotes/snjs/pull/12/commits/efaff05eac248c7c84f83a3fbb532dc0a336914a
Because it was to cover one specific case that was internal to that class, which is also why I decided to make the method private. I don't feel like it would make much sense to a caller when used as a public method on the item manager.

Reply:
Would it be less confusing to just remove one of the implementations so there is only one source of itemsKeys? I think maybe protocol service should remove its function.

Reply:
Well, I like when there is less indirection going on. However the reason we went with `latestItemsKeys()` was to provide some documentation in reaction to the bug we experienced. Still, you could argue that it wouldn't have prevented the bug anyway.
So I'm in favor of removing it but I don't feel very strongly about the decision. Can I let you make the call here?

Reply:
It definitely is subtle. But it seems like really it should be the public function that should be renamed, not the private one, since public consumption is more liable to misuse. So if it was instead itemManager.itemsKeys that was renamed to latestItemsKeys, that would seem to make more sense. Then the private implementation in protocol service can either be removed or renamed to match the itemsManager name, this way we know 100% that they are the exact same function.

Reply:
Okay, I will rename the public function then. This poses the question of what to do with the other functions in itemManager: https://github.com/standardnotes/snjs/pull/12/files#diff-b92d581b5582e43bac311c8c05b33eb0R152-R168
Should they all be renamed to latestNotes, latestTags, etc?